package.nls.json

```diff
@@ -12,7 +12,7 @@
   "deviceSimulatorExpressExtension.commands.microbit.openSimulator": "[micro:bit] Open Simulator",
   "deviceSimulatorExpressExtension.commands.microbit.newFile": "[micro:bit] New File",
   "deviceSimulatorExpressExtension.configuration.title": "Device Simulator Express configuration",
-  "deviceSimulatorExpressExtension.configuration.properties.configEnvOnChange": "When you change the Python Interpreter, the dependencies for the Device Simulator Express will automatically be configured for the extension dependencies (and the interpreter will auto-change if necessary).",
+  "deviceSimulatorExpressExtension.configuration.properties.configEnvOnChange": "When you change the Python interpreter, the Device Simulator Express will atuomatically configure itself for the required dependencies.",

```

Comment:
'automatically' is misspelled.

Reply:
> LGTM. I noticed that the linter outputs a errors before "activating" the extension in the session (before we point to the venv), do you think we can do something about that?

For this, the pylint configuration is added upon activation, so it's hard to have it recognize it before activating :(. Based on our current structure, pylint doesn't know to look in our outfolder until we specify.